bundles/net.tourbook/src/net/tourbook/importdata/DialogEasyImportConfig.java

```diff
@@ -3393,20 +3340,12 @@ private void enable_IL_Controls() {
 
                   for (final Label label : _lblTT_Speed_TourTypeIcon) {
 
-                     if (isILSelected) {
+                     final Integer speedTTIndex = (Integer) label.getData(DATA_KEY_SPEED_TOUR_TYPE_INDEX);
 
-                        final Integer speedTTIndex = (Integer) label.getData(DATA_KEY_SPEED_TOUR_TYPE_INDEX);
+                     final SpeedTourType speedTT = _selectedIL.speedTourTypes.get(speedTTIndex);
+                     final long tourTypeId = speedTT.tourTypeId;
 
-                        final SpeedTourType speedTT = _selectedIL.speedTourTypes.get(speedTTIndex);
-                        final long tourTypeId = speedTT.tourTypeId;
-
-                        label.setImage(TourTypeImage.getTourTypeImage(tourTypeId));
-
-                     } else {
-
-                        // the disabled image looks very ugly
-                        label.setImage(null);
-                     }
+                     label.setImage(TourTypeImage.getTourTypeImage(tourTypeId));
                   }
```

Comment:
Because if you look closely at the current file, you test `isILSelected` twice but it doesn't seem like it has changed in the meantime. That seems redundant and not efficient to test a boolean twice no ?

![image](https://user-images.githubusercontent.com/25235665/157262115-c7710eee-8804-4c66-a76e-a01623b32d31.png)

Reply:
For me it was easier because it's [SonarLint ](https://www.sonarlint.org/eclipse)that told me about it 😄 

There's also this issue that it underlined:

![image](https://user-images.githubusercontent.com/25235665/157344242-2b683b44-617c-47b8-80f7-18129600aa9c.png)

Reply:
Fixed in https://github.com/mytourbook/mytourbook/commit/f15aafea8c5008c20d948c9f04f34be79051b263